requirements.txt

```diff
@@ -2,3 +2,4 @@ anymarkup
 jsonpointer
 requests
 websocket-client
+GitPython
```

Comment:
at least I think it is only used for the "generate" portion of the code right? so that means it probably won't get used that much compared to the other functionality